tbx/project_styleguide/templates/patterns/molecules/author/author.html

```diff
@@ -14,16 +14,4 @@
 {% if author.role %}
     {{ author.role }}
 {% endif %}
-{% if item.date %}
```

Comment:
I believe Jeth is going to add back dates to the blog posts and listings

Reply:
Update: I see the date has moved to a different template. By the way, it might be worth leaving the template changes to @chris-lawton - or at least check with him if he is planning to keep this author template.

Reply:
I've updated this in the FE branch : )